src/Forms/SearchableDropdownTrait.php

```diff
@@ -450,9 +457,16 @@ public function getSchemaDataDefaults(): array
 
     public function getSchemaStateDefaults(): array
     {
-        $data = parent::getSchemaStateDefaults();
-        $data = $this->updateDataForSchema($data);
-        return $data;
+        $state = [
+            'name' => $this->getName(),
+            'id' => $this->ID(),
+            'value' => $this->getDefaultSchemaValue(),
+            'message' => $this->getSchemaMessage(),
+            'data' => [],
+        ];
+
+        $state = $this->updateDataForSchema($state);
+        return $state;
     }
```

Comment:
Not a lot going on here - just pulled through what's happening in `FormField` (and `SingleSelectField` implementation abstracted into `getDefaultSchemaValue()`) so we can avoid calling `getSource()` whenever we render the react component